docs/HowToUseMultiHeap.md

```diff
@@ -57,3 +58,16 @@ CONFIG_RAM_REGIONx_SIZE="1024,512,2048,512,256,4096"
 CONFIG_RAM_REGIONx_HEAP_INDEX="0,1,1,2,2,3"
 ```
 
+## 4. Supported APIs
+Following APIs support to allocate memory for a specific heap.  
+A header file [mm.h](../os/include/tinyara/mm/mm.h) provides the API descriptions.
```

Comment:
A header file [mm.h] provides following APIs which supports to allocate memory for a specific heap region as shown below: